src/summersplash2022/index.module.css

```diff
@@ -1142,204 +1142,96 @@
+  .heading,
+  .handbookHeaderDeskotop {
+    font-size: 300%;
+    line-height: 200%;
   }
 
-  .newVariantsSVG {
-    width: 70%;
-    height: auto;
+  .handbookHeaderDeskotop {
+    padding: 0px;
   }
 
-  .søkHer,
-  .scroll {
-    font-family: 'Graphik Web';
-    font-style: normal;
-    font-weight: 500;
+  p,
+  .section2Paragraph,
+  .whyWorkAtVariant p,
+  .section6 p {
     font-size: 150%;
-    line-height: 34px;
+    line-height: 150%;
   }
 
-  .eller {
-    font-family: 'Graphik Web';
-    font-style: normal;
-    font-weight: 400;
+  /* Section 1 */
+  .section1 p {
     font-size: 150%;
-    line-height: 34px;
-  }
-
-  /* Section 2 styling*/
-  .section2 {
-    width: 80%;
-    color: white;
-  }
-
-  .section2Heading {
-    margin-bottom: 20px;
-    font-family: 'Recoleta';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 200%;
-    line-height: 59px;
+    line-height: 150%;
   }
 
-  .blogpostLink {
-    color: white;
-    text-decoration: underline;
-  }
-
-  .section2Color {
-    background-color: #1a1837;
+  .moreInfoDiv {
+    display: none;
   }
 
-  .section2Paragraph {
-    font-family: 'Graphik Web';
-    font-style: normal;
-    width: 50%;
-    font-size: 110%;
-    line-height: 140%;
+  .imageBlob1 {
+    position: relative;
+    top: 400px;
+    width: 40%;
   }
 
-  .officeMap {
+  .pinkBlob {
     position: relative;
-    width: 70%;
-    height: fit-content;
-    display: flex;
-    right: 13%;
-    padding-top: 13%;
+    width: 40%;
+    bottom: 900px;
   }
 
-  .officeMapMobile {
-    display: none;
+  /* Section 2 */
+  .officeMapSVG {
+    width: 50%;
+    height: auto;
   }
 
   /* Section 3 */
-  .summerstudent1Paragraph,
-  .summerstudent2Paragraph,
-  .summerstudent3Paragraph,
-  .summerstudentName,
-  .summerstudentName1,
-  .summerstudentName2 {
-    font-family: 'Graphik Web';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 110%;
-    line-height: 140%;
-    position: relative;
-    height: fit-content;
-  }
 
-  .summerstudentName,
-  .summerstudentName1,
-  .summerstudentName2 {
-    font-family: 'Recoleta';
-  }
-
-  /* Section 4 */
-  .positionTextLeft {
-    padding-top: 1%;
-    width: 500px;
-    font-family: 'Graphik Web';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 110%;
-    line-height: 140%;
+  .section3 p {
+    font-size: 150%;
+    line-height: 150%;
   }
 
-  .handbookHeaderDeskotop {
-    font-family: 'Recoleta';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 200%;
-    line-height: 39px;
+  .section3 {
+    height: 3000px;
   }
-
-  .handbookParagraphs {
-    width: 500px;
-    font-family: 'Graphik Web';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 110%;
-    line-height: 140%;
+  .summerstudent1picture {
+    width: 20%;
   }
 
-  .readMoreArrow {
+  .greenBlob {
+    width: 50%;
+    height: auto;
     position: relative;
-    right: 20%;
-    top: 250px;
+    right: 300px;
+    bottom: 2500px;
```

Comment:
Usikker på hva som er det eksakte formålet med plasseringen her, så er ikke sikkert det er så relevant, men et godt tips dersom du vil plassere noe etter "midtlinjen" til elementet er å bruke kombinasjonen 
```css
top: 50%;
transform: translateY(-50%); 
```
(eller selvfølgelig med f.eks. left og translateY og andre tall, alt etter hva man ønsker å komme frem til.

Om man gjør det som dette vil midtlinjen til elementet stille seg på 50%, i stedet for at man trenger å kødde rundt med top/bottom for å finne ut av akkurat hvordan man plasserer ting riktig for forskjellige skjermstørrelser.

Dette er forøvrig også et "til neste gang"-tips, trenger ikke bruke det her med mindre det hjelper med å løse noen av issuesa jeg nevnte tidligere liksom 😅